LunarVim/LunarVim
# fix(lsp) launch servers in path as well

# Description



fixes #3366 

## How Has This Been Tested?




- Open a file for which you have installed a language server (without mason)
- See that the language server's setup runs and it starts
